scripts/upgrade

```diff
@@ -32,10 +32,10 @@ if [ -f /etc/nginx/conf.d/${domain}.d/vpnadmin.conf ]; then
   mv /etc/nginx/conf.d/${domain}.d/vpnadmin.conf /etc/nginx/conf.d/${domain}.d/${app}.conf
 fi
 
-if [ -f /etc/php/7.0/fpm/pool.d/vpnadmin.conf ]; then
-  ynh_replace_string "/var/www/vpnadmin/" "/var/www/${app}/" /etc/php/7.0/fpm/pool.d/vpnadmin.conf
-  ynh_replace_string "vpnadmin.sock" "${app}.sock"  /etc/php/7.0/fpm/pool.d/vpnadmin.conf
-  mv /etc/php/7.0/fpm/pool.d/vpnadmin.conf /etc/php/7.0/fpm/pool.d/${app}.conf
```

Comment:
I don't know why we need this lignes to replace the conf naming. Aren't they recreated later in the script with `vpnclient_deploy_files_and_services` function?

Reply:
Hmmm now that you mention it yes it seems odd and unecessary

Beware though that this is about handling legacy stuff from back in the days™ when the conf file was pool.d/vpnadmin.conf instead of pool.d/$app.conf ...

So the ynh_replace_string (and mv) are a bit pointless considering the file gets regenerated later *but* we still want to get rid of the old vpnadmin.conf file somehow.

So maybe something like : 

```
rm -f /etc/php/*/fpm/pool.d/vpnadmin.conf 
```

and similarly for the nginx conf a few lines before (same story, legacy stuff, will be regenerated during ynh_add_nginx_config) : 

```
rm /etc/nginx/conf.d/${domain}.d/vpnadmin.conf
```